release-notes/server-releases.mdx

```diff
@@ -11,6 +11,15 @@ This page includes release notes for supported releases of W&B Server.
 
 <EolReminder/>
 
+<Update label="v0.76.1" description="December 3, 2025">
+**December 3, 2025**
+
+## Fixes
+- Fixed a bug where importing a line plot from a workspace to a report could get stuck if extreme outliers were hidden in the plot.
+- Fixed a bug that could prevent a table panel from resizing correctly if its height was reduced.
+- Fixed a bug where mask overlays could overflow an image with a caption.
```

Comment:
```suggestion
- Fixed a bug where mask overlays could overflow an image with a caption.
- When a client cancels a GraphQL operation, it is now correctly reported as a `4xx` HTTP client error rather than a `5xx` HTTP server error. 
```